elements/pl-dataframe/pl-dataframe.mustache

```diff
@@ -26,7 +26,15 @@
       </div>
 
       <div id="code-{{uuid}}" role="tabpanel" class="tab-pane">
-        <pl-code language="python" copy-code-button="True">{{{code_string}}}</pl-code>
+<!-- Don't unindent this, will cause formatting issues -->
```

Comment:
Yeah, I thought there was a reason for some of these elements were always left-aligned.